### PR TITLE
media viewer version update

### PR DESCRIPTION
**Pull request by olusegz07**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   <div style='position:relative'>
 
-    <div [id]="createElementId('fileInputWrapper')" (click)="fileSelectEvent()"></div>
+    <div [id]="createElementId('fileInputWrapper')" (click)="fileSelectEvent()" (keyup)="fileSelectEvent()"></div>
```
Comment (RiteshHMCTS) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.html:15]: @olusegz07 Is this change require?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="let event of events" (click)="select(event)" [ngClass]="{'EventLogTable-Selected': selected === event}" tabindex="0" attr.aria-label="{{getAriaLabelforRow(event)}}">
+  <tr *ngFor="let event of events" (click)="select(event)" (keyup)="select(event)" [ngClass]="{'EventLogTable-Selected': selected === event}" tabindex="0" attr.aria-label="{{getAriaLabelforRow(event)}}">
```
Comment (RiteshHMCTS) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html:11]: same as above

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.html (hunk 1 above)
Comment (olusegz07) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.html:15]: Yes, it fixes the issue raised by sonarCloud

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html (hunk 2 above)
Comment (olusegz07) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html:11]: Yes, it fixes the issue raised by sonarCloud

**Review by RiteshHMCTS — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html (hunk 2 above)
Comment (andywilkinshmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html:11]: Sonar accessibility check doesn't like it if you don't provide a keyboard equivalent for a mouse action, quite correctly

**Review by andywilkinshmcts — APPROVED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Review by kasi-subramaniam — APPROVED** [review]

**Pull request merged**